src/classes/Callable_API.cls

```diff
@@ -95,6 +95,10 @@ global with sharing class Callable_API implements System.Callable {
                 // parameter service depending on which action was invoked
                 return CallableDispatchService_ERR.getInstanceOf(paramService).dispatch();
 
+            } when 'settings.enableenhancedrecurringdonations' {
+                Boolean scheduleJobsWhenComplete = paramService.getBoolean(CallableApiParameters.PARAM_SCHEDULE_JOBS, true);
+                return  RD2_EnablementDelegate_CTRL.callableApiEnableEnhancement(scheduleJobsWhenComplete);
```

Comment:
Is there a reason to return the queueable job Id from the method?

Reply:
I thought that callableAPI should return something after success, in this case, there is no batch job id to return. If it's not necessary to return anything, will change it.

Reply:
It can return `null`.